openapi.yaml

```diff
@@ -1023,6 +1141,372 @@ paths:
+        start/end period.  
+
+        Clients SHOULD perform Conditional `GET` using the `If-None-Match` header
+        and the `ETag` of a previous `GET` response.
+      tags:
+        - Temporary Ratings
+      parameters:
+        - $ref: '#/components/parameters/period-start-query'
+        - $ref: '#/components/parameters/period-end'
+        - $ref: '#/components/parameters/monitoring-set-filter'
+        - $ref: '#/components/parameters/segment-filter'
+        - $ref: '#/components/parameters/geographical-region-filter'
+        - $ref: '#/components/parameters/subgeographical-region-filter'        
+        - $ref: '#/components/parameters/naming-scheme'
+        - $ref: '#/components/parameters/limit-units'                 
+      responses: 
+        "200":
+          description: OK
+          content:
+            application/vnd.trolie.temporary-rating-set.v1+json:
+              schema:
+                $ref: '#/components/schemas/temporary-rating-set'
+              example:
+                _links:
+                  self:
+                    href: /temporary-ratings
+                total-count: 1
+                offset: 0
+                _embedded:
+                  temporary-ratings:
+                  - _links:
+                      self: 
+                        href: /temporary-ratings/46f7212b-1633-4c30-ba71-c6e987b2ded7
+                      segment: 
+                        href: /reference-data/network-segments/segmentX
+                        name: segmentX
+                    id: 46f7212b-1633-4c30-ba71-c6e987b2ded7
+                    segment-id: segmentX      
+                    start-time: "2025-07-12T16:00:00-07:00"
+                    end-time:  "2025-07-13T12:00:00-07:00"     
+                    updated-time: "2025-07-08T13:05:43.044267100-07:00"
+                    values:
+                    - type: normal
+                      value:
+                        mva: 160
+                    - type: emergency
+                      value:
+                        mva: 165
+                    - type: loadshed
+                      value:
+                        mva: 170  
+                    temporary-aar-exception: true
+                    temporary-seasonal-rating: false
+                    reason: Free-form text reason.  
+            application/json:
+              schema:
+                $ref: '#/components/schemas/temporary-rating-set'                                          
+          headers:
+            X-Rate-Limit-Limit:
+              $ref: '#/components/headers/X-Rate-Limit-Limit'
+            X-Rate-Limit-Remaining:
+              $ref: '#/components/headers/X-Rate-Limit-Remaining'
+            X-Rate-Limit-Reset:
+              $ref: '#/components/headers/X-Rate-Limit-Reset'
+            X-TROLIE-Limit-Units:
+              $ref: '#/components/headers/X-TROLIE-Limit-Units'
+            X-TROLIE-Naming-Scheme:
```

Comment:
Basically the same comment as above. I'm realizing that this PR isn't the place to hash it out, but this is important.

The identity of the segments known to that ratings provider can be specified using the CIM-style names as we discussed previously:
>However, MISO would require the TO to send `"facility-names":[{"name":"RICKROLL_LN_1","authority":"MISO","mrid":"..."}]`  Other names could be included by them. Again, I think this can be addressed by including this naming affordance in the media types instead of the `X-` header on the external submission side.

In other words, both parties would need to know the primary name used by the other ahead of time.

I don't see how this can be a runtime behavior, so a header just doesn't make sense to me.

Reply:
Removed from API until we can readdress with wider group.